### ISSUE TITLE
Multiline Phonetic Guides for alternate zema

### ISSUE THREAD

**Issue by UltimateWishub** [geezorg/emufi#27]
Some zemas have more than one way of saying it. There is also አቋቋም and ዝማሜ which have milikits for ጽፋት, መረግድ, ቁም, and እጅ ሥራ I was wondering if there was a way to add the ድርብ and the others without the inconvenient way of just adding a bunch of text boxes.

**Comment by dyacob** [comment]
Hi David, layout and formatting of miliket goes beyond what EMUFI can accomplish. The text layout limitation is set by whatever word processor (like MS Word, or LibreOffice Writer) or text presentation system (like HTML+CSS in a browser, LaTeX) is capable of.  Most systems will be capable of layout for a _single_ level using built-in support for Ruby annotation, but not do multilevel annotation (except Firefox).  Going to multilevel, there really is no solution that will be portable to all web browsers, or word processors.  The W3C is aware of the issue though and may address it in a CSS revision, then the OpenDocument Format and Word may also follow, but not soon.

So with Word, its basically text boxes above the first level.  However, people read and sing only one level at a time anyway, so you could repeat the zaima text 4 times separately, each wth a single level for ጽፋት, መረግድ, ቁም, and እጅ ሥራ.  If you are working in HTML, you could use the CSS layers, which each single level is on a separate layer, then one layer is shown and 3 others hidden until toggled.  This would be great on a tablet, I don't know if Word can support layers also.  The approach doesn't help the problem on paper (back to text boxes).

Below is an image from my work that show what I think you are describing (created with PowerPoint and lots of boxes):


<img width="1662" alt="Screen Shot 2022-09-18 at 8 16 54 PM" src="https://user-images.githubusercontent.com/8098333/190934189-b9e5f794-2036-47a2-8a6a-c6fd2f09ec62.png">

**Comment by UltimateWishub** [comment]
Ok got it. Can you make video on how you did what you did in the image?

**Comment by dyacob** [comment]
Did you find these videos?  https://youtu.be/FXWNOJTmeyA  This link is to the first of 3, but they address only a single level.  I'm overdue to make a follow up to show the textbox approach, and applying the EMUFI extensions.  Maybe in October I can do that.

**Comment by UltimateWishub** [comment]
Yeah I watched those videos. Your future work will be greatly appreciated.
